### DISCUSSION TITLE
Square size

### DISCUSSION THREAD

**Discussion by aaronlsmiles** [decadenza/SimpleStereo#12] [Q&A] [answered]
What is the `square size` parameter and what does it relate to? 

In the documentation the default = 1, but in the 'examples' it is set to 60.5.

I assumed it was the size of each square on my checkerboard, but I'm confused if this is supposed to be mm or cm, since 1 and 60.5 are so drastically different.

Cheers!

**Comment by decadenza** [comment, marked as answer]
Square size is the square side length in world units. If you use mm, your measures will be in mm (or you can use any other world unit).
It is important that this parameter is correct as all the measures will depend from it. If it is not correct, the 3D reconstruction will be wrong by a certain error factor.

I will update the documentation with a more descriptive definition to avoid any doubts.

**Reply by aaronlsmiles** [reply]
Just so my understanding is clear; 

- Is that the square side length of an individual square, or the size of the entire side of the checkerboard? 

- And when you say; “you can use any other world unit”, if mm; 1 would = 1mm (or 0.1cm), and the 60.5 would = 60.5mm (or 6.05cm)?

**Comment by decadenza** [comment]
Yes, is the side length of a single square of the chessboard.

Yes, the unit you use determines the final measures. If you have a square length of 60.5 mm, you can choose to use "60.5" in mm, or also, "6.05" in cm or even "2.38189" in inches!
Of course, all the measures will be in that unit thereafter.

**Reply by aaronlsmiles** [reply]
Thanks! But why is the example 60.5? That seems like a very large square size. That's what confused me so much heh.

**Reply by decadenza** [reply]
Because that's the size I used. It depends on wide your camera lens is and on the distance you are calibrating from.
This is not one-size-fits-all world.
Cheers.

**Reply by aaronlsmiles** [reply]
Wow, so the size of an individual square on your checkerboard was over 6cm!? That must of been printed larger than A4, right?

**Reply by decadenza** [reply]
Yes. A big laminated board almost 1x1 m   :-)
